tests/probes/yamlfilecontent/test_probes_yamlfilecontent_types.xml

```diff
@@ -0,0 +1,421 @@
+      </criteria>
+    </definition>
+
+   <definition class="compliance" version="1" id="oval:0:def:13">
+      <metadata>
+        <title></title>
+        <description></description>
+      </metadata>
+      <criteria operator="AND">
+        <criterion comment="comment" test_ref="oval:0:tst:13"/>
+      </criteria>
+    </definition>
+
+
+   <definition class="compliance" version="1" id="oval:0:def:14">
+      <metadata>
+        <title></title>
+        <description></description>
+      </metadata>
+      <criteria operator="AND">
+        <criterion comment="comment" test_ref="oval:0:tst:14"/>
+      </criteria>
+    </definition>
+
+
+   <definition class="compliance" version="1" id="oval:0:def:15">
+      <metadata>
+        <title></title>
+        <description></description>
+      </metadata>
+      <criteria operator="AND">
+        <criterion comment="comment" test_ref="oval:0:tst:15"/>
+      </criteria>
+    </definition>
+
+   <definition class="compliance" version="1" id="oval:0:def:16">
+      <metadata>
+        <title></title>
+        <description></description>
+      </metadata>
+      <criteria operator="AND">
+        <criterion comment="comment" test_ref="oval:0:tst:16"/>
+      </criteria>
+    </definition>
+
+   <definition class="compliance" version="1" id="oval:0:def:17">
+      <metadata>
+        <title></title>
+        <description></description>
+      </metadata>
+      <criteria operator="AND">
+        <criterion comment="comment" test_ref="oval:0:tst:17"/>
+      </criteria>
+    </definition>
+
+   <definition class="compliance" version="1" id="oval:0:def:18">
+      <metadata>
+        <title></title>
+        <description></description>
+      </metadata>
+      <criteria operator="AND">
+        <criterion comment="comment" test_ref="oval:0:tst:18"/>
+      </criteria>
+    </definition>
+
+   <definition class="compliance" version="1" id="oval:0:def:19">
+      <metadata>
+        <title></title>
+        <description></description>
+      </metadata>
+      <criteria operator="AND">
+        <criterion comment="comment" test_ref="oval:0:tst:19"/>
+      </criteria>
+    </definition>
+
+   <definition class="compliance" version="1" id="oval:0:def:20">
+      <metadata>
+        <title></title>
+        <description></description>
+      </metadata>
+      <criteria operator="AND">
+        <criterion comment="comment" test_ref="oval:0:tst:20"/>
+      </criteria>
+    </definition>
+
+  </definitions>
+  <tests>
+
+    <ind-def:yamlfilecontent_test version="1" id="oval:0:tst:1" check="all" comment="true">
+      <ind-def:object object_ref="oval:0:obj:1"/>
+    </ind-def:yamlfilecontent_test>
+
+    <ind-def:yamlfilecontent_test version="1" id="oval:0:tst:2" check="all" comment="true">
+      <ind-def:object object_ref="oval:0:obj:2"/>
+    </ind-def:yamlfilecontent_test>
+
+    <ind-def:yamlfilecontent_test version="1" id="oval:0:tst:3" check="all" comment="error">
```

Comment:
Are the comments from the test accurate?
I think all of them should evaluate to `true`, as the object exists and there are no statuses.

Reply:
They aren't accurate, I'll fix that, they all should evaluate to true.